recipe/cmake_test/CMakeLists.txt

```diff
@@ -1,4 +1,45 @@
 project(cf_dummy LANGUAGES C CXX)
 cmake_minimum_required(VERSION 3.12)
+
+option(WITH_TORCH_PYTHON "Build with torch_python" OFF)
+
+find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
+find_package(pybind11 CONFIG REQUIRED)
 find_package(Torch CONFIG REQUIRED)
 find_package(ATen CONFIG REQUIRED)
+
+if(WITH_TORCH_PYTHON)
+  if(WIN32)
+    set(libtorch_base_path $ENV{PREFIX}/Lib/site-packages/torch)
+  else()
+    set(libtorch_base_path $ENV{PREFIX})
+  endif()
+  find_library(torch_python NAMES torch_python HINTS ${libtorch_base_path}/lib/ REQUIRED)
+endif()
+
+set(target_name cmake_test)
```

Comment:
as long as we only have one target, adding a variable for it is not helpful. Please directly use `cmake_test` below